nextflow_schema.json

```diff
@@ -81,17 +81,35 @@
                     "fa_icon": "fas fa-barcode",
                     "description": "Trim barcodes from the output sequences in the FastQ files from basecaller."
                 },
-                "gpu_device": {
+                "kit": {
                     "type": "string",
-                    "default": "auto",
-                    "description": "Device specified in GPU mode using '--device'.",
+                    "default": "",
+                    "description": "sequencing kit'.",
                     "fa_icon": "fas fa-fish"
                 },
-                "gpu_cluster_options": {
+                "flowcell": {
                     "type": "string",
-                    "description": "Cluster options required to use GPU resources (e.g. '--part=gpu --gres=gpu:1').",
+                    "default": "",
+                    "description": "flowcell'.",
                     "fa_icon": "fas fa-fish"
                 },
+                "dorado_device": {
+                    "type": "string",
+                    "default": "cuda:all",
+                    "description": "Device specified using '--device'.",
```

Comment:
Is dorado a particular model of nanopore or something? What is a dorado device?

Reply:
this is for specifying what kind of compute one wants to use: cuda:all for all GPUs or cuda:0 for a specific GPU or CPU